source/Libraries/SteamLibrary/SteamShared/MetadataProvider.cs

```diff
@@ -464,6 +469,69 @@ internal string ParseDescription(string description)
                         break;
                     }
                 }
+
+                var tagNames = tagNamer.GetTagNames();
+                Dictionary<int, string> newTagNames = null;
+
+                var tagKeyValues = metadata.ProductDetails["common"]["store_tags"]?.Children;
+                logger.Debug($"Setting tags for {appId}, found {tagKeyValues?.Count}");
+                if (tagKeyValues != null)
+                {
+                    metadata.Tags = new HashSet<MetadataProperty>();
+                    if (settings.LimitTagsToFixedAmount)
+                    {
+                        tagKeyValues = tagKeyValues.Take(settings.FixedTagCount).ToList();
+                    }
+
+                    foreach (var tag in tagKeyValues)
+                    {
+                        if (int.TryParse(tag.Value, out int tagId))
+                        {
+                            if (settings.BlacklistedTags.Contains(tagId))
+                            {
+                                continue;
+                            }
+
+                            if (!tagNames.TryGetValue(tagId, out string name))
+                            {
+                                if (newTagNames == null)
+                                {
+                                    logger.Debug($"Tag {tagId} not found. Fetching new ones.");
+                                    newTagNames = tagNamer.UpdateAndGetTagNames();
+                                }
+                                if (!newTagNames.TryGetValue(tagId, out name))
+                                {
+                                    logger.Warn($"Could not find tag name for tag {tagId}");
+                                    continue;
+                                }
+                            }
+                            name = tagNamer.GetFinalTagName(name);
+                            metadata.Tags.Add(new MetadataNameProperty(name));
+                        }
+                    }
+                }
+            }
+
+            string parentStr = metadata.ProductDetails?["common"]["parent"]?.Value;

```

Comment:
This is copy paste of already existing code and prevents plugin from building...